MapboxCoreNavigation/EventDetails.swift

```diff
@@ -167,9 +180,9 @@ struct NavigationEventDetails: EventDetails {
         }
         percentTimeInPortrait = totalTimeInPortrait + totalTimeInLandscape == 0 ? 100 : Int((totalTimeInPortrait / (totalTimeInPortrait + totalTimeInLandscape)) * 100)
         
-        var totalTimeInForeground = session.timeSpentInForeground
-        var totalTimeInBackground = session.timeSpentInBackground
```

Comment:
Looks like `totalTimeInForeground`, `totalTimeInBackground` and `rating` were not encoded to result dictionary.